package.json

```
@@ -47,13 +47,16 @@
   "devDependencies": {
     "@brightspace-ui/stylelint-config": "^1",
     "@brightspace-ui/testing": "^1",
+    "@eslint/compat": "^1.2.1",
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.13.0",
```

Let's just use the major version unless we 100% need a minor/patch -- and even then I tend not to care when it's a `devDependency` since we can guarantee it with the lock file anyway.

```suggestion
    "@eslint/compat": "^1",
    "@eslint/eslintrc": "^3",
    "@eslint/js": "^9",
```